packages/runtime-core/__tests__/componentProps.spec.ts

```diff
@@ -275,6 +275,28 @@ describe('component props', () => {
     expect(root.innerHTML).toBe('<div id="b">2</div>')
   })
 
+  test('validator arguments', async () => {
```

Comment:
This file looks like the right place but can't see any tests relating to prop validation, nor elsewhere apart from type tests.

Reply:
Also can't find if there's a type test for the arguments of the `validator`?